framework/src/security/security_internal.h

```diff
@@ -22,6 +22,11 @@
 #include <string.h>
 #include <tinyara/seclink.h>
 
+#if (defined(CONFIG_ARCH_BOARD_ARTIK053S) || defined(CONFIG_ARCH_BOARD_ARTIK053)) \
+		&& defined(CONFIG_S5J_HAVE_SSS)
+#include "../../../os/arch/arm/src/s5j/sss/isp_custom.h"
```

Comment:
Same as above.

Reply:
In future, It will be removed.
But currently, It needs.